src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -85,6 +86,14 @@ public function getConfigTreeBuilder()
                 ->scalarNode('ide')->defaultNull()->end()
                 ->booleanNode('test')->end()
                 ->scalarNode('default_locale')->defaultValue('en')->end()
+                ->booleanNode('use_language_headers')
```

Comment:
should it be `use_language_headers` or `use_language_header` ?

Reply:
also, the name might not be explicit enough to make it clear that this is about setting the locale.

Reply:
`use_language_negotiation`, `use_language_content_negotiation`?

Reply:
Now using 2 options: `set_locale_from_accept_language` and `set_content_language_from_locale`, should be way more explicit and flexible